core/templates/dev/head/domain/suggestion/SuggestionThreadObjectFactorySpec.ts

```diff
@@ -76,79 +66,45 @@ describe('Suggestion thread object factory', function() {
       },
       last_updated: 1000
     };
-  });
-
-  it('should create a new suggestion thread from a backend dict.', function() {
-    suggestionBackendDict.suggestion_type = 'edit_exploration_state_content';
-
-    var suggestionThread = (
-      SuggestionThreadObjectFactory.createFromBackendDicts(
-        suggestionThreadBackendDict, suggestionBackendDict));
-
-    expect(suggestionThread.lastUpdated).toEqual(1000);
-    expect(suggestionThread.originalAuthorName).toEqual('author');
+    let suggestionThread = suggestionThreadObjectFactory.createFromBackendDicts(
+      suggestionThreadBackendDict, suggestionBackendDict);
     expect(suggestionThread.status).toEqual('accepted');
     expect(suggestionThread.subject).toEqual('sample subject');
     expect(suggestionThread.summary).toEqual('sample summary');
+    expect(suggestionThread.originalAuthorName).toEqual('author');
+    expect(suggestionThread.lastUpdated).toEqual(1000);
     expect(suggestionThread.messageCount).toEqual(10);
     expect(suggestionThread.threadId).toEqual('exploration.exp1.thread1');
-
-    var suggestion = suggestionThread.getSuggestion();
```

Comment:
You can still do this and then not change the tests right? That way the getSuggestion() function also gets tested

Reply:
Oh wow :P seems like the test cases have been updated after I opened up the PR. I'll fix them.

Reply:
Done